app/Lib/lang.php

```diff
@@ -79,6 +79,10 @@
   'ct.cmp_enrollment_configurations.pl' => 'CMP Enrollment Configurations',
   'ct.co_dashboard_widgets.1' => 'Dashboard Widget',
   'ct.co_dashboard_widgets.pl' => 'Dashboard Widgets',
+  'ct.co_dashboard_widgets.footer' =>  'Widget Footer',
+  'ct.co_dashboard_widgets.footer.desc' =>  'Text placed below the widget',
+  'ct.co_dashboard_widgets.header' =>  'Widget Header',
+  'ct.co_dashboard_widgets.header.desc' =>  'Text placed below the title and above the widget',
```

Comment:
These are not controller localizations and so should not use the `ct` domain.

Reply:
Done - moved to the "fd." listing.